specification/gp-connect-prescriptions-management-fhir.yaml

```diff
@@ -140,7 +140,7 @@ info:
     
     |Environment|Base URL|
     |-----------|--------|
-    |Sandbox|`https://sandbox.api.service.nhs.uk/gp-connect/patient-facing/prescriptions/FHIR/R4`|
+    |Sandbox|`https://sandbox.api.service.nhs.uk/gp-connect-prescriptions-management-fhir/FHIR/R4`|
```

Comment:
We should not use the `FHIR/R4` suffix here. It should be similar in pattern to access-record. Even if this work now, later when we deploy mock-reciever, the path-suffix won't be sent to mock-reciever which will result in 404s.

Reply:
Removed them [here](98516be64afc85f19c830d0cf0dce0d6e0c9e4c0)